domain-classes-generator/src/main/scala/flatgraph/codegen/DomainClassesGenerator.scala

```diff
@@ -517,6 +479,23 @@ class DomainClassesGenerator(schema: Schema) {
         "\n",
         "\n res\n}"
       )
+
+      val propertyNames = nodeType.properties match {
+        case properties if properties.nonEmpty =>
+          "Node properties:" :: properties.map(property => camelCaseCaps(property.name)).toList
+        case _ => Nil
+      }
+      val containedNodeNames = nodeType.containedNodes match {
```

Comment:
there's a lot of spaghetti code in this class, maybe it's worth extracting things like this into something like 
```scala
def commentForProperty[A](prefix: String, propertyNames: Seq[String]): String = ???
```
and then 

```scala
val nodePropertiesComments = 
  commentForProperty("Node properties:", noteType.properties.map(_.property.name) ++ 
  commentForProperty("Contained properties:", containedNodeNames.map(_.nodeType.name)
```

or so..